tests/components/google_assistant/test_google_assistant.py

```diff
@@ -227,47 +205,39 @@ def test_query_climate_request(hass_fixture, assistant_client):
     body = yield from result.json()
     assert body.get('requestId') == reqid
     devices = body['payload']['devices']
-    assert devices == {
-        'climate.heatpump': {
-            'on': True,
-            'online': True,
-            'thermostatTemperatureSetpoint': 20.0,
-            'thermostatTemperatureAmbient': 25.0,
-            'thermostatMode': 'heat',
-        },
-        'climate.ecobee': {
-            'on': True,
-            'online': True,
-            'thermostatTemperatureSetpointHigh': 24,
-            'thermostatTemperatureAmbient': 23,
-            'thermostatMode': 'heat',
-            'thermostatTemperatureSetpointLow': 21
-        },
-        'climate.hvac': {
-            'on': True,
-            'online': True,
-            'thermostatTemperatureSetpoint': 21,
-            'thermostatTemperatureAmbient': 22,
-            'thermostatMode': 'cool',
-            'thermostatHumidityAmbient': 54,
-        },
-        'sensor.outside_temperature': {
-            'on': True,
-            'online': True,
-            'thermostatTemperatureAmbient': 15.6
-        },
-        'sensor.outside_humidity': {
-            'on': True,
-            'online': True,
-            'thermostatHumidityAmbient': 54.0
-        }
+    assert len(devices) == 3
+    assert devices['climate.heatpump'] == {
```

Comment:
Same tests except broken up in testing per device. That way the error logs are more readable.

Also removed the 'on' attribute as that is part of the OnOff trait that is not enabled for climate devices.